packages/system-data/package.json

```
@@ -0,0 +1,34 @@
+{
+	"name": "@directus/system-data",
+	"version": "0.0.0",
+	"description": "Definitions and types for Directus system collections",
+	"homepage": "https://directus.io",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/directus/directus.git",
+		"directory": "packages/system-data"
+	},
+	"funding": "https://github.com/directus/directus?sponsor=1",
+	"license": "MIT",
+	"author": "Tim de Heiden <tim@directus.io>",
+	"type": "module",
+	"exports": {
+		".": "./dist/index.js",
+		"./package.json": "./package.json"
+	},
+	"main": "dist/index.js",
+	"files": [
+		"dist"
+	],
+	"scripts": {
+		"build": "NODE_ENV=production tsup",
+		"dev": "NODE_ENV=production tsup"
```

Does this need to be `NODE_ENV=production` for dev or should we set it to `development`?
```suggestion
		"build": "NODE_ENV=production tsup",
		"dev": "NODE_ENV=development tsup"
```